pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -190,6 +190,7 @@ func (c *Cluster) makeMgrDaemonContainer(mgrConfig *mgrConfig) v1.Container {
 		SecurityContext: controller.PodSecurityContext(),
 		StartupProbe:    controller.GenerateStartupProbeExecDaemon(config.MgrType, mgrConfig.DaemonID),
 		LivenessProbe:   controller.GenerateLivenessProbeExecDaemon(config.MgrType, mgrConfig.DaemonID),
+		ReadinessProbe:  controller.GenerateMgrReadinessProbeExecDaemon(config.MgrType, mgrConfig.DaemonID),
```

Comment:
cannot be a common name as `GenerateReadinessProbeExecDaemon`?

Reply:
Actually that used to be the name of the function in the first implemenation. But then after the reviews we decided to make this function specefic to the mgr (so I added `Mgr`). In case we add readiness probe for more deamons in the future then we can make this function more generic.